pkg/endpoints/utils.go

```diff
@@ -107,3 +112,45 @@ func (r Resource) RegisterReadinessProbes(container *restful.Container) {
 
 	container.Add(wsv4)
 }
+
+// Store all types here that are reused throughout files
+type Extension struct {
+	Name string
+	Url  string
```

Comment:
URL to make the go linter happy

The comment for this struct is too generic for me, let's include information about the Extension itself?